corehq/apps/app_manager/models.py

```diff
@@ -2392,7 +2392,7 @@ class CaseSearch(DocumentSchema):
     """
     Properties and search command label
     """
-    command_label = DictProperty(default={'en': 'Search All Cases'})
+    command_label = DictProperty(default=ugettext_lazy('Search All Cases'))
```

Comment:
This is setting the default to a string and not a dictionary. though farid can give more context of how this is used